test/unit/array.spec.ts

```diff
@@ -39,16 +39,21 @@ export class ArrayTests {
         Expect(result).toBe(5);
     }
 
+    @TestCase("firstElement()", 3)
+    @TestCase("name", "array")
```

Comment:
Add a testcase for `length` here (since that's what you added)

Reply:
good point